warehouse/query-core/src/main/java/datawave/query/scheduler/PushdownScheduler.java

```diff
@@ -75,9 +75,9 @@
     protected List<Function<IteratorSetting,IteratorSetting>> customizedFunctionList;
     
     /**
-     * Local instance of the table ID
+     * Local instance of the table name
      */
-    protected String tableId;
+    protected String tableName;
```

Comment:
It looks like this member variable isn't being used--could be replaced with a local variable.